src/main/java/core/basesyntax/serviceimpl/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriterImpl implements FileWriterService {
+    @Override
+    public void writeFile(String fileName, String data) {
```

Comment:
According to the checklist, avoid hardcoding in your solution. It's better to pass the path to the file as a parameter to the method. Consider refactoring the method to accept a Path object instead of a String for the file name to make it more flexible and to enforce the use of relative paths.